### ISSUE TITLE
Large UVL models not loading

### ISSUE THREAD

**Issue by MathieuVavrille** [FeatureIDE/FeatureIDE#1358]
### Issue

I tried to open in FeatureIDE a large model in the UVL format, that can be found here https://github.com/Universal-Variability-Language/uvl-models/blob/main/Feature_Models/Operating_Systems/KConfig/embtoolkit.uvl.
After a while eclipse did not respond anymore.
I am on the last release of FeatureIDE.

### Source of the issue (I think)

I tested uvl-parser from https://github.com/neominik/uvl-parser (before it was forked) and I realised that it could take a very long time to load even a small model. However I don't know Clojure so I could not dig deeper into the code. As a comparison, a naive implementation in java (without all the genericity of the language) only need few milliseconds to load the model.
I did not try the new fork of uvl-parser.

Best regards.

**Comment by st-vi** [comment]
tl;dr
By closing the "Feature Model Constraint" Window at the bottom, the UVL model can be loaded in several minutes.

This is not necessarily a problem with UVL, but with large feature models in general. The UVL parser needs a few seconds, but the main problem is showing the feature model. In some cases, closing the outline can help when opening large models. This special model has a disproportionate number of constraints, which leads to the problem when loading it. By closing the "Feature Model Constraint" Window at the bottom, the loading speed can be increased significantly. I can open the model in about 2 - 3 minutes on my machine, with the  "Feature Model Constraint" Window closed.
